parsec/class/parsec_object.h

```diff
@@ -311,25 +317,20 @@ static inline parsec_object_t *parsec_obj_new_debug(parsec_class_t* type, const
         assert(NULL != ((parsec_object_t *) (object))->obj_class);        \
         assert(PARSEC_OBJ_MAGIC_ID == ((parsec_object_t *) (object))->obj_magic_id); \
         if (0 == parsec_obj_update((parsec_object_t *) (object), -1)) {     \
-            parsec_obj_run_destructors((parsec_object_t *) (object));       \
-            PARSEC_OBJ_SET_MAGIC_ID((object), 0);                      \
-            PARSEC_OBJ_REMEMBER_FILE_AND_LINENO( object, __FILE__, __LINE__ ); \
```

Comment:
It's a pity loosing that debugging ability, why can't we keep this here? 

I'd say we don't need the parsec_obj_release function, and just call object->obj_release(object) where the free was

Reply:
> I'd say we don't need the parsec_obj_release function, and just call object->obj_release(object) where the free was

Done.